liwords-ui/src/socket/socket.ts

```diff
@@ -107,10 +109,26 @@ export const LiwordsSocket = (props: {
           userID: decoded.uid,
           loggedIn: decoded.a,
           connID: cid,
+          isChild: decoded.cs,
           path: pathname,
           perms: decoded.perms?.split(','),
         },
       });
+      if (
+        parseInt(decoded.cs) === 2 &&
+        (!localStorage?.getItem('birthdateWarning') ||
+          Date.now() - parseInt(localStorage?.getItem('birthdateWarning')!) >
+            24 * 3600)
```

Comment:
isn't `Date.now()` in milliseconds?
(also it'd be preferable not to `localStorage.getItem` twice)